docs/README.md

```diff
@@ -2,40 +2,43 @@
 
 ![Crossplane](media/banner.png)
 
-Crossplane is an open source Kubernetes add-on that extends any cluster with
-the ability to provision and manage cloud infrastructure, services, and
-applications using kubectl, GitOps, or any tool that works with the Kubernetes
-API.
+Crossplane, a [Cloud Native Computing Foundation][CNCF] sandbox project, is an
+open source Kubernetes add-on that extends any cluster with the ability to
+provision and manage infrastructure and services using kubectl, GitOps, or any
+tool that works with the Kubernetes API.
 
 With Crossplane you can:
 
-* **Provision & manage cloud infrastructure with kubectl**
-  * [Install Crossplane] to provision and manage cloud infrastructure and
-    services from any Kubernetes cluster.
-  * Provision infrastructure primitives from any provider ([GCP], [AWS],
-    [Azure], [Alibaba], on-prem) and use them alongside existing application
-    configurations.
-  * Version, manage, and deploy with your favorite tools and workflows that
-    you’re using with your clusters today.
+* **Compose Custom APIs**
```

Comment:
@jbw976 I agree that this needs to be worded better, and maybe switching provisioning infrastructure to be first would be correct. However, the reasoning for putting composition first is because it matches how we have been moving towards introducing the project in the documentation. As you can see with `v1.1` docs, the [Provisioning Infrastructure](https://crossplane.io/docs/v1.1/getting-started/provision-infrastructure.html) section uses composition, rather than managed resources directly, which is a reflection of the fact that we think most Crossplane users should never create managed resources except through XRs / XRCs. That being said, we are still provisioning infrastructure, just at a higher abstraction level - I wonder if there is a way to lead in the `README.md` with provisioning, but at the higher level as well?